app/views/feedback/new.html.haml

```diff
@@ -0,0 +1,6 @@
+%section.form_wrapper
+  = simple_form_for(@form, url: feedback_path) do |f|
+    = f.hidden_field :url
+    = f.input :email, as: :email
+    = f.input :comment, as: :text
+    = f.button :submit, 'Send Feedback', role: 'button'
```

Comment:
This page has the following issues:
* No h1
* No `class="content-main"` -> required for the proper button formatting to be applied
* No `#content` element -> required for "Skip to main content" link to work

Note: The `section` layout provides the last two but the `application` layout does not. Of course you can't use the section layout anyway but that may be why it's not doing what you expect.

You may wish to add the feedback page(s) to the accessibility tests by adding an `include_examples` to the bottom of `accessibility_spec.rb` to pick some of this up.